code/modules/power/smes.dm

```diff
@@ -67,8 +67,17 @@
 	var/IO = 0
 	var/MC = 0
 	var/C
+	var/tierfive = FALSE
 	for(var/obj/item/stock_parts/capacitor/CP in component_parts)
 		IO += CP.rating
+		if(CP.rating == 5)
+			tierfive = TRUE
+	if(tierfive)
+		IO *= 4 // Quadruples power input/output caps
+		if(powernet) // Uncaps the shock damage from the powernet we're attached to
+			powernet.lethal = TRUE
```

Comment:
what about when we get disconnected from the powernet? you need to keep track of it to make sure you're always connected to the one that has lethal = TRUE when you disconnect/connect to a powernet